jujugui/static/gui/src/app/components/terminal/terminal.js

```diff
@@ -166,9 +166,18 @@ class Terminal extends React.Component {
       if (this.term) {
         this.term.fit();
       }
+      this.focus();
```

Comment:
I believe you'll want this in the above conditional as there will be no textarea if there is no terminal.

Reply:
Done.